lib/cb/utils/validator.rb

```diff
@@ -65,7 +65,13 @@ def try_parse_xml(body)
       end
 
       def error_message(processed_response)
-        processed_response.fetch('errors', processed_response.fetch('Errors', ''))
+        return '' if processed_response.empty?
+        find_errors_node(processed_response) || ''
+      end
+
+      def find_errors_node(processed_response)
+        nested_hash = processed_response[processed_response.keys.first]
```

Comment:
I was trying to check for the requirements. I don't think there's any requirement, so an API could return just a top level errors node.

I guess what we have is fine.